postgres0.yml

```diff
@@ -48,17 +48,16 @@ bootstrap:
 #        archive_command: mkdir -p ../wal_archive && test ! -f ../wal_archive/%f && cp %p ../wal_archive/%f
 #      recovery_conf:
 #        restore_command: cp ../wal_archive/%f %p
+      pg_hba:  # Add following lines to pg_hba.conf after running 'initdb'
```

Comment:
Ok, understood. I see 2 ways to solve this.

- Introduce `pg_hba.base.conf` and include it at the start of pg_hba.conf
- Add standard HBA configuration lines also into DCS.
```
      pg_hba:  # Set pg_hba.conf
        - local   all             all                                     trust
        - host    all             all             127.0.0.1/32            trust
        - host    all             all             ::1/128                 trust
        - local   replication     all                                     trust
        - host    replication     all             127.0.0.1/32            trust
        - host    replication     all             ::1/128                 trust
        - host    replication     replicator      127.0.0.1/32            md5
        - host    all             all             0.0.0.0/0               md5
#        - hostssl all             all             0.0.0.0/0               md5
```

Which one could be better approach? Otherwise, any other ideas?

Reply:
Include won't really work.